.github/workflows/docs-teardown.yml

```
@@ -0,0 +1,23 @@
+#  copy this workflow into your repo
+name: "Documentation Teardown"
+
+on:
+  pull_request_target:
+    branches:
+      - "main"
+    types:
+      - closed
+
+jobs:
+  teardown-docs:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/setup-node@v3
+        with:
+          node-version: lts/*
+      - name: Teardown documentation
+        run: |
+          npm install -g surge
+          surge teardown ${{ github.event.repository.owner.login}}-${{ github.event.repository.name}}-${{ github.event.pull_request.number }}.surge.sh --token ${{ secrets.SURGE_TOKEN }}
+          
```

```suggestion
      - name: Teardown documentation
        run: |
          npm install -g surge
          surge teardown ${{ github.event.repository.owner.login}}-${{ github.event.repository.name}}-${{ github.event.pull_request.number }}.surge.sh --token ${{ secrets.SURGE_TOKEN }}
          ```suggestion
      - name: Comment on PR
        uses: marocchino/sticky-pull-request-comment@v2
        with:
          number: ${{ github.event.pull_request.number }}
          message: |
            Thanks for the documentation updates.

            The preview documentation has now been torn down - reopening this PR will republish it.
          GITHUB_TOKEN: ${{ secrets.PR_COMMENT_TOKEN }}
```
```